### ISSUE TITLE
Compiling MFEM in parallel fails 

### ISSUE THREAD

**Issue by salazardetroya** [mfem/mfem#1264]
In my macOS 10.14.6, using openMPI 1.8.5 with clang-1001.0.46.4, I build the MFEM in parallel with hypre-2.10.0b and metis 4.0. When linking the examples, I get this error:

```
[ 39%] Linking CXX executable ex1
cd /Users/salazardetro1/scicomp_libraries/mfem/mfem/build/examples && /usr/local/Cellar/cmake/3.15.3/bin/cmake -E cmake_link_script CMakeFiles/ex1.dir/link.txt --verbose=1
/Library/Developer/CommandLineTools/usr/bin/c++  -O3 -DNDEBUG -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  CMakeFiles/ex1.dir/ex1.cpp.o  -o ex1 ../libmfem.a /Users/salazardetro1/scicomp_libraries/mfem/metis-4.0/libmetis.a
 "mfem::GradientIntegrator::AssemblePA(mfem::FiniteElementSpace const&, mfem::FiniteElementSpace const&)", referenced from:
      vtable for mfem::GradientIntegrator in libmfem.a(bilininteg.cpp.o)
  "mfem::VectorDivergenceIntegrator::AssemblePA(mfem::FiniteElementSpace const&, mfem::FiniteElementSpace const&)", referenced from:
      vtable for mfem::VectorDivergenceIntegrator in libmfem.a(bilininteg.cpp.o)
  "mfem::GradientIntegrator::AddMultTransposePA(mfem::Vector const&, mfem::Vector&) const", referenced from:
      vtable for mfem::GradientIntegrator in libmfem.a(bilininteg.cpp.o)
  "mfem::GradientIntegrator::AddMultPA(mfem::Vector const&, mfem::Vector&) const", referenced from:
      vtable for mfem::GradientIntegrator in libmfem.a(bilininteg.cpp.o)
  "mfem::VectorDivergenceIntegrator::AddMultTransposePA(mfem::Vector const&, mfem::Vector&) const", referenced from:
      vtable for mfem::VectorDivergenceIntegrator in libmfem.a(bilininteg.cpp.o)
  "mfem::VectorDivergenceIntegrator::AddMultPA(mfem::Vector const&, mfem::Vector&) const", referenced from:
      vtable for mfem::VectorDivergenceIntegrator in libmfem.a(bilininteg.cpp.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed 
exit code 1 (use -v to see invocation)
```

Do I need some external library to habilitate PA? Maybe CEED? The linker is c++, but I enabled MPI, maybe that is the problem? Thanks.

Edit: Adding the symbols output as suggested by Geoff [symbols_output.txt](https://github.com/mfem/mfem/files/4119291/symbols_output.txt), also included the whole output that I forgot earlier (similar messages)

**Comment by tzanio** [comment]
@jandrej, can you take a look?

**Comment by goxberry** [comment]
@salazardetroya It might help to include the output of
```bash
nm ${path_to}/libmfem.a | c++filt
```
where `${path_to}` is the absolute directory path containing `libmfem.a`. (A suitable relative path would also work.) The output probably won't mean much to me, but might be useful for MFEM developers because it will expose the symbols provided/required by the library.

It would also be worth including any configuration files you needed to write/generate so that other people can try to reproduce the error you're seeing.

**Comment by adrienbernede** [comment]
Solved by #1265.
